elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient.scala

```diff
@@ -453,6 +453,8 @@ object RestlasticSearchClient {
 
     case class ElasticErrorResponse(error: JValue, status: Int) extends Exception(s"ElasticsearchError(status=$status): ${error.toString}")
 
+    case class DeleteByQuerySearchResponse(deletedDocumentsCount: BigInt)
```

Comment:
Let's use regular `Long` (any reason why `BigInt`?)

Reply:
Or you can use `count.asInstanceOf[BigInt].toLong` I hope

Reply:
Done